rn/Teacher/ios/Teacher/Localizable.xcstrings

```diff
@@ -9993,6 +9996,9 @@
           }
         }
       }
+    },
+    "Files ((count))" : {
+
```

Comment:
Maybe the branch is not in sync with the master but we shouldn't have this here. The formatting is invalid, it should be `Files (%lld)`.